src/test/regress/expected/undistribute_table.out

```diff
@@ -0,0 +1,327 @@
+SELECT * FROM seq_table ORDER BY a;
+ id | a
+---------------------------------------------------------------------
+  5 | 1
+  9 | 2
+  3 | 3
+(3 rows)
+
+SELECT undistribute_table('seq_table');
+ undistribute_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT objid::regclass AS "Sequence Name" FROM pg_depend WHERE refobjid = 'seq_table'::regclass::oid AND classid = 'pg_class'::regclass::oid;
+  Sequence Name
+---------------------------------------------------------------------
+ seq_table_a_seq
+(1 row)
+
+INSERT INTO seq_table (id) VALUES (7), (1), (8);
+SELECT * FROM seq_table ORDER BY a;
+ id | a
+---------------------------------------------------------------------
+  5 | 1
+  9 | 2
+  3 | 3
+  7 | 4
+  1 | 5
+  8 | 6
+(6 rows)
+
+DROP TABLE seq_table;
+--test tables with views
+CREATE TABLE view_table (a int, b int, c int);
+SELECT create_distributed_table('view_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO view_table VALUES (1, 2, 3), (2, 4, 6), (3, 6, 9);
+CREATE VIEW undis_view1 AS SELECT a, b FROM view_table;
+CREATE VIEW undis_view2 AS SELECT a, c FROM view_table;
+CREATE VIEW undis_view3 AS SELECT b, c FROM undis_view1 JOIN undis_view2 ON undis_view1.a = undis_view2.a;
+SELECT * FROM pg_views WHERE viewname LIKE 'undis\_view%';
+     schemaname     |  viewname   | viewowner |                          definition
+---------------------------------------------------------------------
+ undistribute_table | undis_view1 | postgres  |  SELECT view_table.a,                                       +
+                    |             |           |     view_table.b                                            +
+                    |             |           |    FROM view_table;
+ undistribute_table | undis_view2 | postgres  |  SELECT view_table.a,                                       +
+                    |             |           |     view_table.c                                            +
+                    |             |           |    FROM view_table;
+ undistribute_table | undis_view3 | postgres  |  SELECT undis_view1.b,                                      +
+                    |             |           |     undis_view2.c                                           +
+                    |             |           |    FROM (undis_view1                                        +
+                    |             |           |      JOIN undis_view2 ON ((undis_view1.a = undis_view2.a)));
+(3 rows)
+
+SELECT * FROM undis_view3 ORDER BY 1, 2;
+ b | c
+---------------------------------------------------------------------
+ 2 | 3
+ 4 | 6
+ 6 | 9
+(3 rows)
+
+SELECT undistribute_table('view_table');
+ undistribute_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT * FROM pg_views WHERE viewname LIKE 'undis\_view%';
```

Comment:
it might be useful to explicitly list the columns in case it changes in future PG versions